### DISCUSSION TITLE
Relative widget positioning

### DISCUSSION THREAD

**Discussion by dbrennan** [bluecaret/carettab#360] [Ideas and Feature Requests]
Have you considered the option to support relative widget position based on either the previous widget or a selected widget?
If that latter you'd have to deal with circular references, but just being relative to the preceding widget would be really helpful.

**Comment by bluecaret** [comment]
That's an interesting concept. I think it could potentially complicate the UI more than it already is but is definitely something I'll have to give some thought to. Thanks for the suggestion!
